.swiftlint.yml

```diff
@@ -0,0 +1,12 @@
+opt_in_rules:
+  - sorted_imports
+  - trailing_closure
+
+file_length:
+  ignore_comment_only_lines: true
+
+disabled_rules:
+  - trailing_whitespace
```

Comment:
Could we remove this? I think leaving this enabled to warn of trailing whitespace is a fine idea.

Reply:
yeah, it's annoying how Xcode sometimes handles this, which is why I added it. Personally I like things tidy as well, so I'll fix up all these places too.